src/BeSimple/SoapBundle/DependencyInjection/Configuration.php

```diff
@@ -95,6 +95,8 @@ private function addClientSection(ArrayNodeDefinition $rootNode)
                             ->children()
                                 ->scalarNode('wsdl')->isRequired()->end()
                                 ->scalarNode('user_agent')->end()
+                                ->scalarNode('timeout')->end()
+                                ->scalarNode('connection_timeout')->end()
```

Comment:
https://symfony.com/doc/current/components/config/definition.html#node-type
Are you sure you don't want the integer type for these?

Although the other ones also seemingly depend on this to just be the correct type.

Reply:
I'll add them here with a minimum of `0`.